### PR TITLE
PMM-7143 add latest value for section RAM in block NodeSummary

### PR DESCRIPTION
**Pull request by anton-bystrov**
Recreate branch from main and fix the bug with memory.
Link for issue - https://jira.percona.com/browse/PMM-7143
137314893-e99f2b1e-7269-45e7-83ad-d3fd9deef45c
137314903-a4d0ed5e-d8d4-4dd2-ab48-656f0554cc4d
Fix issue with memory in MongoDB cluster
mongos:
изображение
rs:
изображение
cfg:
изображение
изображение

### CODE REVIEW

**Review by atymchuk — APPROVED** [review]

**Review by tiagomotasantos — APPROVED** [review]

**Review by tiagomotasantos — DISMISSED** [review]

File: dashboards/MySQL/MySQL_Instance_Summary.json [hunk 1]
```diff
@@ -6206,15 +6281,12 @@
         "notice": false,
         "now": true,
         "refresh_intervals": [
+            "1s",
```
Comment (atymchuk) [inline dashboards/MySQL/MySQL_Instance_Summary.json:6284]: Let's double check the `1s` interval. @puneet0191 What do you think?

Comment (puneet0191) [reply dashboards/MySQL/MySQL_Instance_Summary.json:6284]: @anton-bystrov we have default 1m as refresh interval on most of the dashboards, why use 1s here?

Comment (anton-bystrov) [reply dashboards/MySQL/MySQL_Instance_Summary.json:6284]: Here I can't say why interval 1s. I'll change it to 1m.

Comment (anton-bystrov) [reply dashboards/MySQL/MySQL_Instance_Summary.json]: I removed interval 1 second at all dashboards.

Comment (atymchuk) [reply dashboards/MySQL/MySQL_Instance_Summary.json]: I'd definitely leave it to the discretion of QA team. We need to test and confirm that this change is fine.

**Review by atymchuk — APPROVED** [review]

**Review by atymchuk — APPROVED** [review]
LGTM; Will we mention the removal of 1s time intervals in our docs?
@rasika-chivate PTAL

**Pull request closed without merging**